plugins/chat/assets/javascripts/discourse/services/chat-notification-manager.js

```diff
@@ -159,7 +161,10 @@ export default class ChatNotificationManager extends Service {
 
     if (this.site.desktopView) {
       const channel = await this.fetchChannel(data.channel_id);
-      data.isDirectMessageChannel = channel.isDirectMessageChannel ?? false;
+
+      if (channel) {
+        data.isDirectMessageChannel = channel.isDirectMessageChannel ?? false;
```

Comment:
This is used by the chat audio initializer, it shouldn't have much impact if it is not set, especially since we use a logical not operator when checking for this value so it equates to a true/false value.